CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Bump version for lodash, y18n, and ssri dependencies 
+- Mark `getObservableVideoMetrics` optional in ClientMetricReprt and `videoStreamIndex` and `selfAttendeeId` optional in DefaultClientMetricReport
```

Comment:
Nit: Can we backtick `ClientMetricReprt` and `DefaultClientMetricReport` and end this log with a "."?

Reply:
Sure, will do.